WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -1542,13 +1542,44 @@ public void onFailure(ApiHelper.ErrorType errorType, String errorMessage, Throwa
         task.execute(apiArgs);
     }
 
-    private void trackAddMediaEvents(boolean isVideo, boolean fromMediaLibrary) {
+    /**
+     * Analytics about new media
+     *
+     * @param isVideo Whether is a video or not
+     * @param uri The URI of the media on the device, or null
+     * @param path The path of the media on the device, or null
+     */
+    private void trackAddMediaFromDevice(boolean isVideo, Uri uri, String path) {
```

Comment:
I think renaming this to `trackAddMediaFromDeviceEvents` would be more descriptive